commons/src/main/java/io/hyscale/commons/commands/provider/ImageCommandProvider.java

```diff
@@ -159,7 +146,7 @@ private String docker() {
         return DOCKER_COMMAND + SPACE;
     }
 
-    private String normalize(String input) {
+    public String normalize(String input) {
```

Comment:
When you are making this public and this method is not related to imagecommandprovider get this out to something like "ImageNameNormalizer"